code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -163,11 +175,24 @@
 	name = "white tie"
 	icon_state = "whitetie"
 
+/obj/item/clothing/accessory/tie/col
```

Comment:
This typepath should be a bit more descriptive. Like:
`/obj/item/clothing/accessory/tie/colorable`